source/GlassView.Core/Models/AllocationInfo.cs

```diff
@@ -0,0 +1,9 @@
+namespace GlassView.Core.Models;
+
+public sealed class AllocationInfo
+{
+    public required Int32 Gen0Collections { get; set; }
```

Comment:
I assume these properties won't change after they were imported, so would it be an idea to use `{ get; init; }`?

Reply:
Good point. I'll check if `System.Text.Json` allows for that.

Reply:
Done. Gladly, `System.Text.Json` allows for `init `setters :-)
A rad side effect is that our models are now immutable :-)